pyqtgraph/opengl/items/GLTextItem.py

```diff
@@ -0,0 +1,74 @@
+from typing import List, Union
+from OpenGL.GL import *
+from OpenGL.GLUT import *
+import numpy as np
+from pyqtgraph.opengl.GLGraphicsItem import GLGraphicsItem
+
+class GLTextItem(GLGraphicsItem):
+    """Draws text in 3D."""
+
+    pos:Union[np.ndarray, List[float]] = np.array([0.0, 0.0, 0.0])
+    color:np.ndarray = np.array([1.0, 1.0, 1.0, 1.0])
+    text:str = ''
```

Comment:
`pos`, `color` and `text` should almost certainly not be class variables, but object properties, so should probably move them into the `__init__` method.

If you were using lists instead of numpy arrays, it may be possible to move the position of all of these TextItems, by mutating the list (vs. just the instance you want to change).

Reply:
I fixed in [e97bd80](https://github.com/pyqtgraph/pyqtgraph/pull/1776/commits/e97bd800ddf16d23729192ce7b6afc9f6549a504).

Reply:
👍🏻